http.go

```diff
@@ -10,15 +10,16 @@ import (
 	"github.com/pkg/errors"
 )
 
+// DefaultMethod uses GET for the default request type
+var DefaultMethod string = "GET"
```

Comment:
This isn't really necessary, but instead of `"GET"` you could do `http.MethodGet`.